src/NuGet.Core/NuGet.DependencyResolver.Core/Remote/RemoteDependencyWalker.cs

```diff
@@ -128,115 +97,148 @@ public async Task<GraphNode<RemoteResolveResult>> WalkAsync(LibraryRange library
             };
 
             Debug.Assert(node.Item != null, "FindLibraryCached should return an unresolved item instead of null");
+            MergeRuntimeDependencies(runtimeDependencies, node);
```

Comment:
Same with this correct?

Reply:
Same. Not intended to be a significant change.